### PR TITLE
[29249] Initialize Project

### PR DESCRIPTION
**Pull request by toannn-2309**
Relative URL: https://edu-redmine.sun-asterisk.vn/issues/29249
Screenshot from 2020-09-15 10-25-59
Database
Screenshot from 2020-09-15 10-10-59

### CODE REVIEW

**Comment by toannn-2309** [comment]
ready

**Comment by toannn-2309** [comment]
ready

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: bỏ hết comment cho sáng sủa a

**Review by saobv-2223 — COMMENTED** [review]
chưa có database.yml.example và credentials.yml.exaple

**Comment by toannn-2309** [comment]
ready

**Comment by toannn-2309** [comment]
ready

**Pull request merged**